internal/handlers/handler_firstfactor.go

```diff
@@ -174,6 +174,7 @@ func FirstFactorPost(msInitialDelay time.Duration, delayEnabled bool) middleware
 		userSession.Groups = userDetails.Groups
 		userSession.Emails = userDetails.Emails
 		userSession.AuthenticationLevel = authentication.OneFactor
+		userSession.Authenticated = time.Now().Unix()
```

Comment:
On full 2FA environments the user would be considered authenticated only after 2FA stage.

Reply:
Fixed.